src/mono/wasm/Makefile

```diff
@@ -33,7 +33,7 @@ all: build-native icu-files source-files header-files
 #  If EMSDK_PATH is not set by the caller, download and setup a local emsdk install.
 #
 
-EMSCRIPTEN_VERSION=`cat emscripten-version.txt`
+EMSCRIPTEN_VERSION:=`cat emscripten-version.txt`
```

Comment:
I ran into another problem with the file read as outlined in the issue here: https://github.com/dotnet/runtime/issues/49862

Sent a PR: https://github.com/dotnet/runtime/pull/49863